.changeset/unlucky-cars-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+'earljs': patch
+---
+
+Remove dead code from mock implementation.
```

Comment:
nit: For now, I don't use changesets for changes not directly affecting users like refactorings etc. The main reason being the inability to specify the lack of impact on API (it's not a `patch` change).